apps/ima/fields/index.js

```diff
@@ -2,9 +2,243 @@
 
 module.exports = {
   'who-are-you': {
-    isPageHeading: true,
     mixin: 'radio-group',
     options: ['person-named', 'has-legal-representative', 'someone-else'],
+    validate: 'required',
+    legend: {
+      className: 'visuallyhidden'
+    }
+  },
+  'in-the-uk': {
+    isPageHeading: true,
+    mixin: 'radio-group',
+    options: ['yes', 'no'],
+    validate: 'required'
+  },
+  name: {
+    labelClassName: 'visuallyhidden',
+    validate: ['required'],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
```

Comment:
Same issue with validation as the helper-details page.

Reply:
done